keyboards/momoka_ergo/readme.md

```diff
@@ -0,0 +1,19 @@
+# momoka_ergo
+
+![momoka_ergo](https://github.com/StefanGrindelwald/TestDemo/blob/master/Momoka_ergo.jpg?raw=true)
+
+*A short description of the keyboard/project*
```

Comment:
?

Reply:
I corrected this mistake. Please review this again. Thank you.